src/shell/shell.cpp

```diff
@@ -1219,19 +1221,19 @@ void SHELL_Init() {
 	        "\n"
 	        "Where:\n"
 	        "  [color=light-cyan]TEXT[reset]         is the text to display as a prompt, or empty.\n"
-	          "  /c[:][color=white]CHOICES[reset] Specifies allowable keys, which default to [color=white]yn[reset].\n"
-	          "  /n           Do not display the choices at end of prompt.\n"
-	          "  /s           Enables case-sensitive choices to be selected.\n"
-	          "  /t[:][color=white]c[reset],[color=light-magenta]nn[reset]    Default choice to [color=white]c[reset] after [color=light-magenta]nn[reset] seconds.\n"
+	          "  /c[:][color=white]CHOICES[reset] specifies allowable keys, which default to [color=white]yn[reset].\n"
+	          "  /n           does not display the choices at end of prompt.\n"
+	          "  /s           enables case-sensitive choices to be selected.\n"
```

Comment:
I have mixed feelings about these. How I see it, we either do one of the following:

- Complete sentences that start with uppercase letter, and then we put a period at the end
- Incomplete sentences that start with a lowercase letter, and there's NO period at the end

Also, please restore the use of the imperative mood. We should use imperative mood all the time (I know currently it's inconsistent, but we should strive towards that goal). So "do not display", "enable case-sensitive", etc.

Study the output of some GNU Linux commands, such as `ls --help`, `cp --help` for guidance.

Reply:
I chose the second option, as most descriptions are short.
For the few which were two sentences I separated them by semicolons, leaving only one sentence, to avoid changing styles